### ISSUE TITLE
Masque compatibility

### ISSUE THREAD

**Issue by Leiludallas** [jordonwow/bigdebuffs#302]
Hi, Can we have a masque compatibility or a way to change the icons skin please?
Great addon btw.

**Comment by anhell** [comment]
Sorry for intruding, but I would really love this.

**Comment by Wolfforcex** [comment]
I was wondering the same, as the icons are way off the portrait and look very out of place. https://freeimage.host/i/HxmcIR9

**Comment by mont3crysto** [comment]
Agree: changing the icon appearance or, at least, removing its border should be a thing. Thanks.
